python/python/lancedb/pydantic.py

```diff
@@ -35,13 +35,13 @@
 import numpy as np
 import pyarrow as pa
 import pydantic
-import semver
+from packaging.version import Version
 
-PYDANTIC_VERSION = semver.parse_version_info(pydantic.__version__)
+PYDANTIC_VERSION = Version(pydantic.__version__)
```

Comment:
This is technically a BC break since this is a public attribute, i.e. `lancedb.pydantic.PYDANTIC_VERSION`. Thoughts?

Reply:
Eh I don't think anyone should be relying on this, and it's easy for them to change their code to not. So I don't feel the need to mark it as a breaking change.